make install error

Hello
I get source normally
./configure
make 

I get an error after make instal
CMake Error at cmake_install.cmake:97 (FILE):
  file INSTALL cannot find
  "/tmp/srt/$<TARGET_FILE_DIR:srt-live-transmit>/stransmit".

The issue is still persistent on centos7 with cmake 2.8

```
Linking CXX shared library libsrt.so
[100%] Built target srt_shared
[ 30%] Built target haicrypt_virtual
[ 74%] Built target srt_virtual
[ 76%] Built target srt_static
[ 89%] Built target srtsupport_virtual
[ 92%] Built target srt-file-transmit
[ 94%] Built target srt-live-transmit
[ 97%] Built target srt-multiplex
[100%] Built target srt_shared
Install the project...
-- Install configuration: "Release"
-- Installing: /tmp/build/bins/ffmpeg_build/lib64/libsrt.so.1.3.2
-- Installing: /tmp/build/bins/ffmpeg_build/lib64/libsrt.so.1
-- Installing: /tmp/build/bins/ffmpeg_build/lib64/libsrt.so
-- Installing: /tmp/build/bins/ffmpeg_build/lib64/libsrt.a
-- Installing: /tmp/build/bins/ffmpeg_build/include/srt/version.h
-- Installing: /tmp/build/bins/ffmpeg_build/include/srt/srt.h
-- Installing: /tmp/build/bins/ffmpeg_build/include/srt/logging_api.h
-- Installing: /tmp/build/bins/ffmpeg_build/include/srt/platform_sys.h
-- Installing: /tmp/build/bins/ffmpeg_build/include/srt/udt.h
-- Installing: /tmp/build/bins/ffmpeg_build/include/srt/srt4udt.h
-- Installing: /tmp/build/bins/ffmpeg_build/lib64/pkgconfig/haisrt.pc
-- Installing: /tmp/build/bins/ffmpeg_build/lib64/pkgconfig/srt.pc
-- Installing: /tmp/build/bins/ffmpeg_build/bin/srt-live-transmit
-- Up-to-date: /tmp/build/bins/ffmpeg_build/bin/srt-live-transmit
CMake Error at cmake_install.cmake:123 (FILE):
  file INSTALL cannot find
  "/tmp/build/bins/ffmpeg_sources/srt/$<TARGET_FILE_DIR:srt-live-transmit>/stransmit".


make: *** [install] Error 1
[user@env dev]$ ll bins/ffmpeg_sources/
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -93,7 +93,7 @@ option(USE_STATIC_LIBSTDCXX "Should use static rather than shared libstdc++" OFF
 option(ENABLE_INET_PTON "Set to OFF to prevent usage of inet_pton when building against modern SDKs while still requiring compatibility with older Windows versions, such as Windows XP, Windows Server 2003 etc." ON)
 option(USE_OPENSSL_PC "Use pkg-config to find OpenSSL libraries" ON)
 
-set(TARGET_srt "srt" CACHE STRING "The name for the haisrt library")
+set(TARGET_srt "srt" CACHE STRING "The name for the SRT library")
 
 if (NOT ENABLE_LOGGING)
 	set (ENABLE_HEAVY_LOGGING OFF)
@@ -708,25 +708,6 @@ if ( ENABLE_CXX11 )
 	# Applications
 
 	srt_add_application(srt-live-transmit ${VIRTUAL_srtsupport})
-
-	# For backward compatibility with the old name
-	if (SYMLINKABLE)
-		set (REPLI_COMMAND create_symlink )
-		# It appears impossible to get the filename component from the generator
-		# expression $<TARGET_FILE:srt-live-transmit>, hence we predict it as:
-		set (srt_live_transmit_name srt-live-transmit${CMAKE_EXECUTABLE_SUFFIX})
-	else()
-		set (REPLI_COMMAND copy)
-		set (srt_live_transmit_name $<TARGET_FILE:srt-live-transmit>)
-	endif()
-
-	set (stransmit_path $<TARGET_FILE_DIR:srt-live-transmit>/stransmit${CMAKE_EXECUTABLE_SUFFIX})
-	add_custom_command(
-		TARGET srt-live-transmit
-		POST_BUILD
-		COMMAND ${CMAKE_COMMAND} -E ${REPLI_COMMAND} ${srt_live_transmit_name} ${stransmit_path})
-	install(FILES ${stransmit_path} DESTINATION ${CMAKE_INSTALL_BINDIR})
-
 	srt_add_application(srt-file-transmit ${VIRTUAL_srtsupport})
 
 	if (MINGW)

--- a/README.md
+++ b/README.md
@@ -22,7 +22,7 @@ As audio/video packets are streamed from a source to a destination device, SRT d
 
 # Guides
 * [Why SRT Was Created](docs/why-srt-was-created.md)
-* [Using the `stransmit` App](docs/stransmit.md)
+* [Using the `srt-live-transmit` and `srt-file-transmit` Apps](docs/stransmit.md)
 * [SRT Encryption](docs/encryption.md)
 * [API](docs/API.md)
 * [Reporting problems](docs/reporting.md)

--- a/docs/stransmit.md
+++ b/docs/stransmit.md
@@ -1,17 +1,20 @@
-Stransmit
+SRT Live Transmit
 ---------
 
-The *stransmit* tool is a universal data transport tool, which's
+The *srt-live-transmit* tool is a universal data transport tool, which's
 intention is to transport data between SRT and other medium.
+At the same time it is just a sample application to show some of 
+the powerful features of SRT. We encourage you to use SRT library
+itself integrated into your products.
 
 It can be both used as a universal SRT-to-something-else flipper, as
 well as a testing tool for SRT.
 
 The general usage is the following:
 
-    stransmit <input-uri> <output-uri> [options]
+    srt-live-transmit <input-uri> <output-uri> [options]
 
-The following medium types are handled by stransmit:
+The following medium types are handled by *srt-live-transmit*:
 
 - SRT - use SRT for reading or writing, in listener, caller or rendezvous mode, with possibly additional parameters
 - UDP - read or write the given UDP address (also multicast)
@@ -39,6 +42,8 @@ if its form is not directly the IPv4 address.
 
 Medium: FILE (including standard process pipes)
 -----------------------------------------------
+
+**NB!** File mode, except `file://con` is supported in *srt-file-transmit* tool!
 
 The general syntax is: `file:///global/path/to/the/file`. No
 parameters in the URL are extracted. There's one (non-standard!)
@@ -90,7 +95,7 @@ options that can be set through the parameters:
 SRT can be connected using one of three connection modes:
 
 - **caller**: the "agent" (this application) sends the connection request to the peer, which must be **listener**, and this way it initiates the connection.
-- **listener**: the "agent" waits for being contacted by any peer **caller** (note that a listener can accept multiple callers, but *stransmit* does not use this possibility - after the first connected one, it no longer accepts new connections).
+- **listener**: the "agent" waits for being contacted by any peer **caller** (note that a listener can accept multiple callers, but *srt-live-transmit* does not use this possibility - after the first connected one, it no longer accepts new connections).
 - **rendezvous**: A one-to-one only connection where both parties are equivalent and both connect to one another simultaneously. Whoever happened to start first (or succeeded to punch through the firewall) is meant to have initiated the connection.
 
 This mode can be specified explicitly using the **mode** parameter. When it's not specified, then it is "deduced" the following way:
@@ -115,7 +120,7 @@ specified in the URI:
     -   The **adapter** parameter can be used to specify the adapter.
     -   The **port** parameter is not used.
 
-Some parameters handled for SRT medium are specific, all others are socket options. The following parameters are handled special way by *stransmit*:
+Some parameters handled for SRT medium are specific, all others are socket options. The following parameters are handled special way by *srt-live-transmit*:
 
 - **mode**: enforce caller, listener or rendezvous mode
 - **port**: encorce the **outgoing** port (the port number that will be set in the UDP packet as a source port when sent from this host). This can be used only in **caller mode**.
@@ -146,7 +151,7 @@ of this and it sets these options at appropriate time.
 
 Note also that **blocking** option has no practical use for users.
 Normally the non-blocking mode is used only when you have an event-driven application that needs a common
-signal bar for multiple event sources, or you prefer fibers to threads, when working with multiple SRT sockets in one application. The *stransmit* application isn't defined this way. This makes that the practical result of non-blocking mode here is that it uses polling on exactly one socket with infinite timeout. Every reading and writing operation will then return always without blocking, but when they report the "again" situation the application will stall on `srt_epoll_wait()` call. This option then exists for the testing purposes, as well as educational, to serve as an example of how your application should use the non-blocking mode.
+signal bar for multiple event sources, or you prefer fibers to threads, when working with multiple SRT sockets in one application. The *srt-live-transmit* application isn't defined this way. This makes that the practical result of non-blocking mode here is that it uses polling on exactly one socket with infinite timeout. Every reading and writing operation will then return always without blocking, but when they report the "again" situation the application will stall on `srt_epoll_wait()` call. This option then exists for the testing purposes, as well as educational, to serve as an example of how your application should use the non-blocking mode.
 
 
 Command-line Options
@@ -163,7 +168,7 @@ shell (using quotes or backslash).
     - The alarm is set up after the reading loop has started, **not when the application has started**. That is, a caller will still wait the standard timeout to connect, and a listener may wait infinitely until some peer connects; only after the connection is established is the alarm counting started. 
     - **The timeout mechanism doesn't work on Windows at all.** It behaves as if the timeout was set to **-1** and it's not modifiable.
 - **-chunk, -c** - use given size of the buffer. When 0, it uses default 1316, which is the maximum size for a single SRT sending call
-- **-bandwidth, -bitrate, -b** - slow down sending to not send more data than given bandwidth (in bits per second). Note that it has nothing to do with SRT-controlled bandwidth; rather a possibility to have a "live-alike" reading speed from an infinite-speed input source (such as file). Better to think of it as for "testing purposes" because the real live transmission requires its speed synchronized with TS timestamps and `stransmit` has no ability to interpret them.
+- **-bandwidth, -bitrate, -b** - slow down sending to not send more data than given bandwidth (in bits per second). Note that it has nothing to do with SRT-controlled bandwidth; rather a possibility to have a "live-alike" reading speed from an infinite-speed input source (such as file). Better to think of it as for "testing purposes" because the real live transmission requires its speed synchronized with TS timestamps and *srt-live-transmit* has no ability to interpret them.
 - **-verbose, -v** - display additional information on the standard output. Note that it's not allowed to be combined with output specified as **file://con**
 - **-bandwidth-report, -bitrate-report, -report, -r** - Throw periodic bandwidth reports on the standard output. The value is the number of sending events between reports.
 - **-crash, -k** - Force the program to "crash" (terminate abnormally with core dump) when a problem occurred (even such as connection got broken). This is useful only for developers.

